src/components/Tag.tsx

```diff
@@ -0,0 +1,15 @@
+import { FC, PropsWithChildren } from "react";
+import classnames from "classnames";
+
+interface Props {
+  show: boolean;
```

Comment:
```suggestion
  isVisible: boolean;
```
Maybe a little clearer?